src/7-to-8/major-changes/play-pause-stop.rst

```diff
@@ -29,9 +29,34 @@ commands:
 * ``cylc stop``
 
 A workflow can be safely played, paused and stopped any number of times without
-interrupting the workflow. The play command will always pick up where it left
-off. If you want to run again from the start, install a new run or start a
-:term:`reflow`.
+interrupting the workflow.
+
+
+Re-Running Workflows
+--------------------
+
+The ``cylc play`` command will always pick up where it left off (a
+:term:`restart`).
+
+If you want to re-run the entire workflow again from the start either:
+
+* Install a new run.
+* Or if you want to keep the data from the old run start a :term:`reflow`.
+
+It is still possible to re-run workflows in-place in the Cylc 7 manner, however,
+this is discouraged.
```

Comment:
I wonder if a warning box with a wee explanation of why it's discouraged might be apposite here?

Reply:
Considered, however, I wanted to keep this topic brief, it's a minor sidenote.